pkgs/applications/altcoins/steem/default.nix

```
@@ -0,0 +1,99 @@
+{ stdenv, fetchFromGitHub
+, cmake, pkgconfig, autoconf, automake, libtool
+, boost, openssl, ncurses, readline
+, perl, doxygen, python3
+#, zlib, bzip2, snappy # not currently in use; aren't truly needed by lib?
+}:
+let
+
+  version = "0.20.6";
+
+  # These values are embedded into the respective binaries, but don't appear to be crucial.
+  rev = "5d0a4327a22160eea8f1ffe5a7bad7c5ed813ac8";
+  revDate = "Wed Oct 24 10:51:32 2018";
+  sha256 = "1ylhp74myxy2ixlxrj8pfm0vfby15hzzr85ymwmlc0vba4si6in0";
+  fcRev = "EmbeddedInSteemRepo";
+  fcRevDate = revDate;
+
+  pythonJinja = python3.withPackages (pypkgs: [pypkgs.jinja2]);
+
+in stdenv.mkDerivation {
```

```suggestion
in stdenv.mkDerivation rec {
```

so you can self reference `version` within the attr set.